bridge/test/stakingNFT/stakingNFT.test.ts

```diff
@@ -3491,7 +3491,7 @@ contract("StakingNFT", async () => {
       // deposit Token to be distributed
       const scaleFactor = await stakingNFT.getAccumulatorScaleFactor();
       const magic = BigNumber.from("42");
-      const tokenString = "1234567890";
+      const tokenString = "123456789";
```

Comment:
lots of repetition here. Possibly an opportunity for refactoring, but not a huge deal